BuildingBlocks/src/BuildingBlocks.API/Serilog/IdentityAddressMaskingOperator.cs

```diff
@@ -0,0 +1,7 @@
+﻿using Serilog.Enrichers.Sensitive;
+
+namespace Backbone.BuildingBlocks.API.Serilog;
+public class IdentityAddressMaskingOperator : RegexMaskingOperator
+{
+    public IdentityAddressMaskingOperator() : base(@"did:e:[a-zA-Z0-9]+:dids:[a-zA-Z0-9]{22}") { }
```

Comment:
I think you should be able to use the regex we define in the IdentityAddress. For that, it the IdentityAddress class, make the `IdentityAddressValidatorRegex` public and use it here.

Reply:
I did just that and it doesn't "pick up" the addresses.

Reply:
Then you did something wrong, I think. I tested the regex on regex101.com, and it worked.

Reply:
The problem was in the `^` and `&` at the beginning and the end, that prevent it from detecting the addresses in the middle of strings, even though it worked fine with standalone addresses.